.github/workflows/package-release.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           python scripts/check_unreleased_changelog_items.py
           git config --global tag.gpgSign true
       - name: Python Semantic Release
-        uses: python-semantic-release/python-semantic-release@v9.8.6
+        uses: python-semantic-release/python-semantic-release@v9.8.3
```

Comment:
This was verified as working in https://github.com/nfelt14/tm_devices/actions/runs/10083565172